extension/src/background/helpers/dataStorage.ts

```diff
@@ -46,5 +49,9 @@ export const dataStorageAccess = {
     await dataStorage.setItem({ [keyId]: value });
     localStorage.setItem(keyId, value);
   },
+  remoteItem: async (keyId: string) => {
+    await dataStorage.removeItem(keyId);
+    localStorage.removeItem(keyId);
+  },
```

Comment:
Just a heads up, I think you're going to run into some issues when merging with Release/3.0.0

For ex, Release/3.0.0 has no localStorage calls at all now

Sorry for all the headaches rebasing to 3.0.0 is going to cause 😞 I can help you sort through the madness